ecsclient/baseclient.py

```diff
@@ -104,9 +106,15 @@ def remove_cached_token(self):
 
     def _fetch_headers(self):
         token = self.token if self.token else self._token_request.get_token()
-        return {'Accept': 'application/json',
-                'Content-Type': 'application/json',
-                'x-sds-auth-token': token}
+	if self.override_header == None:
+            return {'Accept': 'application/json',
+                    'Content-Type': 'application/json',
+                    'x-sds-auth-token': token}
```

Comment:
how about we make this a variable and add the `X-EMC-Override`... instead of returning the entire block.